Search/Exception/MetadataNotFoundException.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Massive\Bundle\SearchBundle\Search\Exception;
+
+class MetadataNotFoundException extends \RuntimeException
```

Comment:
Haven't heard about SPOT before, but from what I have read there it is not an issue here, since the exception is only for the developer. I would definitely add more information here, but it's not that important to me, since we can add it also later.